mautic/mautic
# [UI] Cycling label colors for different segments in contact details

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴 
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 



## Description

Dependency:
- #13689 

This update addresses the challenge of quickly differentiating between segments in contact details. 

By implementing a cycling color scheme, we're enhancing visual clarity and reducing cognitive load for users who frequently navigate through contact information. 

it also standardize the user experience (as it adds a darker border to all of these elements, indicating visually they’re clickable)





| Before                                 | After
| -------------------------------------- | ---
| ![image](https://github.com/mautic/mautic/assets/116097999/d5ad3034-670c-4c46-ad0c-84693d5d549b)                                       | ![image](https://github.com/mautic/mautic/assets/116097999/dbcaba72-baae-4f97-9bdf-f88cd76e8571)




---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Segments
3. Create 8 segments and save
4. Go to Contacts
5. Create a contact then Save & Close
6. Back to the Contacts list
7. Select the contact checking the checkbox
8. Use the actions dropdown to change their segments
9. Assign all segment to the contact
10. Save
11. Open the contact details again 
12. See the labels
